entries/j/jianguoyun.com.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Site Name": {
+    "domain": "jianguoyun.com",
+    "tfa": [
+      "sms",
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "WeChat Official Account Verification Code"
```

Comment:
`WeChat Official Account Verification Code` sounds off.
On the website it would look like this:
![](https://d.tempfiles.download/D61E0775434AE5/?p=aiQF6qh7RS50TMY4ZD9)

I suggest just using `WeChat` 
```suggestion
      "WeChat"
```